tests/unit/components/status/StatusBar.spec.js

```diff
@@ -84,4 +98,75 @@ describe("StatusWidget.vue", () => {
       "Status:3dbb8814-09f1-44db-b7d5-7a9f702beac4"
     );
   });
+  test("When Vuex is mutated to an ERROR UUID, Then the status text should update as 'Error Occurred' and the the dialog of ErrorCatchWidget is visible ", async () => {
```

Comment:
I don't see any tests for checking that the `/shutdown` route is called when the status mutates to ERROR.  Did I miss it somewhere?

Reply:
Like didn't understand, are you suggesting to have `status: 400`

Reply:
no, a test to check that /shutdown was called.  For example (like in axios_helper and other places): `expect(mocked_axios.history.get[0].url).toStrictEqual(whole_url);`

Reply:
currently I believe if you remove the call in the StatusWidget component to axios that all the tests will still pass

Reply:
Will include the above and few more testcases.